ast/function_declaration_test.go

```diff
@@ -585,14 +587,17 @@ func TestSpecialFunctionDeclaration_MarshalJSON(t *testing.T) {
 					},
 				},
 			},
-			DocString: "test",
-			StartPos:  Position{Offset: 34, Line: 35, Column: 36},
+			Comments: Comments{
+				Leading: []*Comment{NewComment(nil, []byte("///test"))},
+			},
+			StartPos: Position{Offset: 34, Line: 35, Column: 36},
 		},
 	}
 
 	actual, err := json.Marshal(decl)
 	require.NoError(t, err)
 
+	// TODO(preserve-comments): Do we need to include comments in the JSON AST?
```

Comment:
I'd expect the JSON output to have `Comments`, given it was added to the Go struct 🤔

Reply:
It was added to the struct, but I haven't explicitly updated the json serialization logic (MarshalJSON) of AST nodes yet: https://github.com/bartolomej/cadence/blob/14834e4c604b3be9a6f07e25a11876ac9944163e/ast/function_declaration.go#L197-L216